pkg/minikube/bootstrapper/bsutil/testdata/v1.12/containerd-api-port.yaml

```diff
@@ -40,6 +40,7 @@ networking:
 ---
 apiVersion: kubelet.config.k8s.io/v1beta1
 kind: KubeletConfiguration
+cgroupDriver: systemd
```

Comment:
this doesn't seem to be relevent change to this PR

Reply:
I have to set the `cgroupDriver` in the kubelet config file to get rid of this warning in the kubelet log - 

> minikube kubelet[3637]: Flag --cgroup-driver has been deprecated ...

With that change, the tests need to be updated as well.